README.md

```diff
@@ -24,19 +22,27 @@ See [`navapbc/platform`](https://github.com/navapbc/platform) for other template
 
 ## Installation
 
-To get started using the template application on your project, run the following command to execute the [install script](https://github.com/navapbc/template-application-nextjs/tree/main/template-only-bin/install-template.sh), which clones the template repository, copies the template files to your repository, and then cleans up the template repository:
+To get started using the template application on your project:
 
-```bash
-curl https://raw.githubusercontent.com/navapbc/template-application-nextjs/main/template-only-bin/download-and-install-template.sh | bash -s
-```
+1. Run the download and [install script](https://github.com/navapbc/template-application-nextjs/tree/main/template-only-bin/install-template.sh) in your project's root directory.
```

Comment:
did you mean to link to [download-and-install-template.sh](https://github.com/navapbc/template-application-nextjs/blob/main/template-only-bin/download-and-install-template.sh) instead of [install-template.sh](https://github.com/navapbc/template-application-nextjs/blob/main/template-only-bin/install-template.sh)